coding-concepts.js

```diff
@@ -37,23 +37,24 @@ const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:a
+// b) Verify and explain:Ruby-First index was ruby in the array.
 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:SATURDAY SUNDAY
+// b) Verify and explain:error-not a function-shouldn't be in brackets.
```

Comment:
This error occurs because .toUpperCase can only be ran on strings and not arrays.